pubspec.yaml

```diff
@@ -18,5 +18,7 @@ dev_dependencies:
   test: ^1.16.0-nullsafety.4
 
 dependency_overrides:
+  # Because pkg:test does not support pkg:http ^0.13.0
+  test: ^1.16.0-nullsafety.5
```

Comment:
Note that just `^1.16.0-nullsafety` should be sufficient, although this is also likely fine.